Cannot read properties of undefined (reading 'features')

Hello, i have the following error every time i try to download something -

```
[ '.your_mp3_files_will_be_saved_here' ]
Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'features')
    at Object.window.WWebJS.sendMessage (__puppeteer_evaluation_script__:92:39)
    at __puppeteer_evaluation_script__:10:45
    at ExecutionContext._evaluateInternal (C:\Users\u\whatsapp-music-bot\node_modules\puppeteer\src\common\ExecutionContext.ts:273:13)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at ExecutionContext.evaluate (C:\Users\u\whatsapp-music-bot\node_modules\puppeteer\src\common\ExecutionContext.ts:140:12)
    at Client.sendMessage (C:\Users\u\whatsapp-music-bot\node_modules\whatsapp-web.js\src\Client.js:594:28)
[ERROR] 14:13:00 Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'features')
    at Object.window.WWebJS.sendMessage (__puppeteer_evaluation_script__:92:39)
    at __puppeteer_evaluation_script__:10:45
```

--- a/package.json
+++ b/package.json
@@ -13,24 +13,24 @@
     "start": "ts-node-dev --transpile-only --ignore-watch node_modules src/main.ts"
   },
   "devDependencies": {
+    "@types/ffmpeg": "^1.0.4",
     "@types/yt-search": "^2.3.0",
-    "@typescript-eslint/eslint-plugin": "^4.9.0",
-    "@typescript-eslint/parser": "^4.9.0",
-    "eslint": "^7.15.0",
-    "eslint-config-airbnb-base": "^14.2.1",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-prettier": "^3.2.0",
-    "prettier": "^2.2.1",
-    "ts-node-dev": "^1.0.0",
-    "typescript": "^4.1.2",
-    "@types/ffmpeg": "^1.0.4"
+    "@typescript-eslint/eslint-plugin": "^5.28.0",
+    "@typescript-eslint/parser": "^5.28.0",
+    "eslint": "^8.17.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "prettier": "^2.7.0",
+    "ts-node-dev": "^2.0.0",
+    "typescript": "^4.7.3"
   },
   "dependencies": {
     "ffmpeg": "^0.0.4",
     "qrcode-terminal": "^0.12.0",
-    "whatsapp-web.js": "^1.16.5",
-    "yt-search": "^2.10.2",
-    "ytdl-core": "^4.9.1"
+    "whatsapp-web.js": "^1.16.7",
+    "yt-search": "^2.10.3",
+    "ytdl-core": "^4.11.0"
   }
 }

